packages/conf-gtksourceview3/conf-gtksourceview3.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: "The gtksourceview programmers"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://projects.gnome.org/gtksourceview/"
+license: "LGPL 2.1"
+build: [["pkg-config" "--short-errors" "--print-errors" "gtksourceview-3.0"]]
+depends: ["conf-pkg-config" {build}]
+depexts: [
+  ["gtksourceview3-dev"] {os-distribution = "alpine"}
+  ["gtksourceview3"] {os-distribution = "archlinux"}
+  ["epel-release" "gtksourceview3-devel"] {os-distribution = "centos"}
+  ["libgtksourceview-3.0-dev"] {os-distribution = "debian"}
+  ["gtksourceview3-devel"] {os-distribution = "fedora"}
+  ["gtksourceview3"] {os = "freebsd"}
+  ["gtksourceview3"] {os = "openbsd"}
+  ["gtksourceview3-devel"] {os-distribution = "opensuse"}
+  ["libgtksourceview-3.0-dev"] {os-distribution = "ubuntu"}
+  ["gtksourceview3" "libxml2"] {os = "macos" & os-distribution = "homebrew"}
+]
+synopsis: "Virtual package relying on a GtkSourceView-3 system installation"
+description:
+  "This package can only install if libgtksourceview-3.0-dev is installed on the system."
+flags: conf
```

Comment:
Don't worry about linting failures with this flag, they are known and will be fixed in the next opam release. This file looks correct